db/migrate/20211115060245_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
```

Comment:
It is auto-generated by `Devise` but we use `citext` column for storing case insensitive data such as emails, username etc https://nimblehq.co/compass/development/web/database/#data-types

Reply:
Good catch! Fixed in [`3a8b1ec` (#30)](https://github.com/suho/google-search-ruby/pull/30/commits/3a8b1ecffc4b6d94f7ad5903c24c98ec3158850b)